.github/workflows/go-releaser.yml

```diff
@@ -69,10 +69,16 @@ jobs:
         env:
           PACKAGECLOUD_TOKEN: ${{ secrets.IO_PACKAGECLOUD_TOKEN }}
         run: |
+          version="$(echo "${GITHUB_REF}" | sed -e 's|refs/tags/||')";
+          if [[ "$version" =~ '-' ]]; then \
```

Comment:
Same as in https://github.com/timescale/promscale_extension/pull/224#discussion_r863536251, can we have a in-code comment explaining how this selector works?